src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLang.java

```diff
@@ -1,60 +1,81 @@
 package com.wikia.webdriver.testcases.desktop.createawikitests;
 
 import static com.wikia.webdriver.common.core.Assertion.assertEquals;
+import static com.wikia.webdriver.common.core.Assertion.assertStringContains;
 
 import com.wikia.webdriver.common.contentpatterns.CreateWikiMessages;
 import com.wikia.webdriver.common.core.annotations.Execute;
 import com.wikia.webdriver.common.core.api.WikiFactory;
 import com.wikia.webdriver.common.core.helpers.User;
 import com.wikia.webdriver.common.dataprovider.CreateNewWikiDataProvider;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import com.wikia.webdriver.elements.communities.desktop.pages.CreateNewWikiPage;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep1;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep2;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep3;
+
 import org.testng.annotations.Test;
 
 @Test(groups = {"CNW_lang"})
+@Execute(onWikia = "community")
 public class CreateWikiTestsLang extends NewTestTemplate {
 
-  @Test(dataProviderClass = CreateNewWikiDataProvider.class, dataProvider = "getLangs", groups = {
-      "CreateNewWiki_lang_001", "CNW_lang_first"})
-  @Execute(asUser = User.USER_CNW)
-  public void createNewWikiLangTC001(String lang) {
-    WikiBasePageObject base = new WikiBasePageObject();
-    CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
-    cnw1.selectLanguage(lang);
-    String expectedDomainSufix;
-    if (lang.equals("en")) {
-      expectedDomainSufix = String.format("");
-    } else {
+  private void createNewLanguageWiki(String lang) {
+    CreateNewWikiPage cnw = new CreateNewWikiPage()
+        .open()
+        .selectLanguage(lang);
+
+    String expectedDomainSufix = "";
+    if (!lang.equals("en")) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the "en" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1132)